akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -3294,7 +3294,7 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    * of time between events.
    *
    * If you want to be sure that no time interval has no more than specified number of events you need to use
-   * [[throttle()]] with maximumBurst attribute.
+   * [[throttle]] with maximumBurst attribute.
```

Comment:
Different syntax than the `@see` just below?

Reply:
ok, I search and replaced `()]]`, I think it should be `Flow.throttle` to make both IntelliJ and Scaladoc happy. I'll try.

Reply:
Changed to `@see [[throttle]]`, but both seems to work. Entire akka-stream scaladoc would need major cleanup, but that's not for this PR.